doc: Added information about LTE Link Monitor in getting started (#1898)
NordicPlayground/fw-nrfconnect-nrf: doc/nrf/gs_testing.rst

 * 1 stop bit
 * No parity
 * HW flow control: None

.. _lte_connect:

How to connect with LTE Link Monitor
************************************

To connect to nRF9160-based boards (for example, the nRF9160 DK or Nordic Thingy:91), you can also use `LTE Link Monitor`_, which is implemented in `nRF Connect for Desktop`_.
This application is used to establish LTE communication with the nRF9160 modem through AT commands, and it also displays the UART output.

To connect to the nRF9160-based board with LTE Link Monitor, perform the following steps:

1. Launch the LTE Link Monitor app.

   .. note::

      Make sure that **Automatic requests** is enabled in LTE Link Monitor.

#. Connect the nRF9160-based board to the PC with a USB cable.
#. Power on the nRF9160-based board.
#. Click **Select Device** and select the particular board entry from the drop-down list in the LTE Link Monitor.

> #. Observe that the LTE Link monitor app starts AT communication with the modem of the nRF9160-based board and shows the status of the communication in the display terminal.

Reviewer: Thingy:91 does start the application firmware when powering on, but the DK doesn't (at least my 0.7.0 DK doesn't). So it would make sense to mention that in case of the DK, either before or after _select device_ the reset button must be pressed on the DK to start the application (e.g. AT_client).

Author: Adding the following as a note after Select Device and after the initial display of communication: 
In the case of nRF9160 DK, the reset button must be pressed to restart the device and to start the application.

 * 1 stop bit
 * No parity
 * HW flow control: None

.. _lte_connect:

How to connect with LTE Link Monitor
************************************

> To connect to nRF9160 based board devices, you can also use `LTE Link Monitor`_ which is implemented in `nRF Connect for Desktop`_.

Reviewer 2: ```suggestion
To connect to nRF9160-based boards (for example, the nRF9160 DK or Nordic Thingy:91), you can also use `LTE Link Monitor`_, which is implemented in `nRF Connect for Desktop`_.
```
"board devices" sounds a bit strange. Let's just call them boards and give Thingy as an example. Or we could use device instead of board, up to you. :)

"which" needs a comma - see https://www.quickanddirtytips.com/education/grammar/which-versus-that

> This application is used to establish LTE communication with the nRF9160 modem through AT commands.

Reviewer 2: ```suggestion
This application is used to establish LTE communication with the nRF9160 modem through AT commands, and it also displays the UART output.
```

> To connect to the nRF9160 based board device with LTE Link Monitor, perform the following steps:

Reviewer 2: ```suggestion
To connect to the nRF9160-based board with LTE Link Monitor, perform the following steps:
```
Same change for further occurrences.

1. Launch the LTE Link Monitor app.

   .. note::

      Make sure that **Automatic requests** is enabled in LTE Link Monitor.

#. Connect the nRF9160 based board device to the PC with a USB cable.
#. Power on the nRF9160 based board device.
#. Click **Select Device** and select the particular board device entry from the drop-down list in the LTE Link Monitor.

> #. Observe that the LTE Link monitor app starts AT communication with the modem of the nRF9160 based board and shows the status of the communication in the display terminal.

Reviewer 2: Add that it also displays any information that is logged on UART.

>    See the `AT Commands Reference Guide`_ for information on the various AT commands used in the communication.

Reviewer 2: I don't think we need this here, because this topic is more about the UART output - it should stay in the AT client doc.

Author: Removed this.
